diff --git a/drivers/rtc/interface.c.orig b/drivers/rtc/interface.c
index 9d2a263..1153446 100644
--- a/drivers/rtc/interface.c.orig
+++ b/drivers/rtc/interface.c
@@ -442,29 +442,6 @@ static int __rtc_set_alarm(struct rtc_device *rtc, struct rtc_wkalrm *alarm)
 	else
 		err = rtc->ops->set_alarm(rtc->dev.parent, alarm);
 
-	/*
-	 * Check for potential race described above. If the waiting for next
-	 * second, and the second just ticked since the check above, either
-	 *
-	 * 1) It ticked after the alarm was set, and an alarm irq should be
-	 *    generated.
-	 *
-	 * 2) It ticked before the alarm was set, and alarm irq most likely will
-	 * not be generated.
-	 *
-	 * While we cannot easily check for which of these two scenarios we
-	 * are in, we can return -ETIME to signal that the timer has already
-	 * expired, which is true in both cases.
-	 */
-	if ((scheduled - now) <= 1) {
-		err = __rtc_read_time(rtc, &tm);
-		if (err)
-			return err;
-		now = rtc_tm_to_time64(&tm);
-		if (scheduled <= now)
-			return -ETIME;
-	}
-
 	trace_rtc_set_alarm(rtc_tm_to_time64(&alarm->time), err);
 	return err;
 }
